packages/types/src/interfaces/contracts/runtime.ts

```diff
@@ -24,7 +24,7 @@ export const runtime: DefinitionsCall = {
             },
             {
               name: 'gasLimit',
-              type: 'u64'
+              type: 'Weight'
```

Comment:
This is an option: https://github.com/paritytech/substrate/blob/78f8f708781da6fd340ef2a1fc288060a35295ad/frame/contracts/src/lib.rs#L1215